tests/translator/output/httpapi_with_propagate_tags.json

```diff
@@ -0,0 +1,157 @@
+{
+  "Resources": {
+    "ApiGatewayDomainNameV25fe29fe649": {
+      "Properties": {
+        "DomainName": "admin.one.amazon.com",
+        "DomainNameConfigurations": [
+          {
+            "CertificateArn": "arn::cert::abc",
+            "EndpointType": "REGIONAL"
+          }
+        ],
+        "Tags": [
+          {
+            "Key": "TagKey1",
+            "Value": "Value1"
+          },
+          {
+            "Key": "TagKey2",
+            "Value": "Value2"
+          },
+          {
+            "Key": "httpapi:createdBy",
+            "Value": "SAM"
+          }
+        ]
+      },
+      "Type": "AWS::ApiGatewayV2::DomainName"
+    },
+    "MyApi": {
+      "Properties": {
+        "Body": {
+          "info": {
+            "title": {
+              "Ref": "AWS::StackName"
+            },
+            "version": "1.0"
+          },
+          "openapi": "3.0.1",
+          "paths": {},
+          "tags": [
+            {
+              "name": "TagKey1",
+              "x-amazon-apigateway-tag-value": "Value1"
+            },
+            {
+              "name": "TagKey2",
+              "x-amazon-apigateway-tag-value": "Value2"
+            },
+            {
+              "name": "httpapi:createdBy",
+              "x-amazon-apigateway-tag-value": "SAM"
+            }
+          ],
+          "x-amazon-apigateway-cors": {
+            "allowOrigins": [
+              "*"
+            ]
+          }
+        },
+        "FailOnWarnings": true,
+        "Tags": [
+          {
+            "Key": "TagKey1",
+            "Value": "Value1"
+          },
```

Comment:
Surprised `cfn-lint` didn't catch this since this shouldn't even deploy. 🤔

Reply:
ha-ha `cfn-lint` probably doesn't check because it's unstandardized mess

Reply:
Seems like we need non-blocking integration tests for each SAM resource with Tags otherwise, never can be sure

Reply:
But it should be correctly in the schema already that `cfn-lint` uses to check format. 😢